usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -48,6 +51,16 @@
         "custom_schema": "recipient_hash STRING, federal_accounts STRING, cfdas ARRAY<STRING>,"
         " tas_components ARRAY<STRING>",
     },
+    "sam_recipient": {
+        "model": DUNS,
+        "source_table": "duns",
```

Comment:
A `source_query` is necessary to run the `load_query_to_delta` command. This should be the query that transforms the table from it's raw broker form to the intermediary needed by later ETLs. It should apply the same transformations as the query in `restock_duns.sql`

Reply:
Added and called sam_recipient_sql_string